ml-data-hub-plugin/src/test/groovy/com/marklogic/gradle/task/BaseTest.groovy

```diff
@@ -144,6 +148,19 @@ class BaseTest extends Specification {
         DocumentBuilder builder = factory.newDocumentBuilder()
         return builder.parse(new File("src/test/resources/" + resourceName).getAbsoluteFile())
     }
+	
+	protected InputStream getResourceStream(String resourceName) {
+		return BaseTest.class.getClassLoader().getResourceAsStream(resourceName)
+	}
+	
+	protected JSONObject getJsonResource(String fileLoc) {
```

Comment:
use jackson's API rather tha JSONObject here.  The type is JsonNode and we already use it everywhere.